Miraheze/Mirahezelocalset.php

```diff
@@ -405,6 +405,16 @@
 			'superadmin' => array(
 				'admin',
 			),
```

Comment:
When you add this here it prevents the quantix wiki array above from being closed. Add another ), in the right spot above